pkg/expr/func_num.go

```diff
@@ -112,3 +115,24 @@ func average(v ...interface{}) float64 {
 
 	return sum / j
 }
+
+func random(v ...float64) (out float64, err error) {
+	var from, to float64
+
+	switch len(v) {
+	case 1:
+		if to = v[0]; to < 0 {
+			return 0, errors.New("unexpected input type")
+		}
+	case 2:
+		if from, to = v[0], v[1]; from < 0 || to < 0 {
+			return 0, errors.New("unexpected input type")
+		}
+	default:
+		return 0, errors.New("insufficient input")
+	}
+
+	rand.Seed(time.Now().UnixNano())
```

Comment:
I think that seeding is already done somewhere in initialization; try to see if you can find instance, else ignore this comment.